x/gov/bucket.go

```diff
@@ -101,6 +101,51 @@ func (b *ProposalBucket) Update(db weave.KVStore, id []byte, obj *Proposal) erro
 	return nil
 }
 
+const (
+	indexNameElectorate = "electorate"
+)
+
+// ResolutionBucket is the persistence bucket for resolutions.
+type ResolutionBucket struct {
+	orm.IDGenBucket
+}
+
+func NewResolutionBucket() *ResolutionBucket {
+	b := migration.NewBucket(packageName, "resolution", orm.NewSimpleObj(nil, &Resolution{})).
+		WithIndex(indexNameAuthor, indexElectorate, false)
```

Comment:
I think you wanted to change it here... indexNameElectorate

Reply:
Yeah, const definitions that are not grouped on top of the file confuse me, so I think I just changed the wrong reference. The good thing is it will all pop up in tests. Thanks for catching that!